### DISCUSSION TITLE
Which 🇪🇺 country has bicycles outnumbering people?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#740] [Galaxy Trivia] [answered]
**Bonus:** Which city has the most cyclists?

**Comment by SANKARAMDAS** [comment]
The Netherlands lives up to its cyclist-friendly reputation and has the most bicycles per capita of any country in the world. With 90% of it's 16.8 million population being regular cyclists, it's no surprise that the Netherlands is considered the most popular place for cycling. 20% of all trips are many in the country, while it has boasted the highest number of cyclists per capita for over a century. 

Utrecht, Netherlands, is ranked as the most bicycle-friendly city in the world. According to the index, the city counts about 51 %of the population as cyclists.

**Comment by peterbaccaro** [comment, marked as answer]
The **Netherlands** is the European Union (EU) country where bicycles outnumber people. The country is well-known for its cycling culture, and it is estimated that there are over 22 million bicycles in the Netherlands, which is more than the country's population of around 17 million people. Cycling is a popular mode of transportation in the Netherlands, with a well-developed network of cycling paths, bike-friendly cities, and bike parking facilities.

While the Netherlands is the country with the most cyclists per capita, the city with the most cyclists is actually **Copenhagen, Denmark**.
